src/coreclr/src/zap/zapinfo.cpp

```diff
@@ -450,7 +450,7 @@ void ZapInfo::CompileMethod()
     }
 #endif
 
-#if defined(TARGET_X86) || defined(TARGET_AMD64)
+#if defined(TARGET_X86) || defined(TARGET_AMD64) || defined(TARGET_ARM64)
     if (methodAttribs & CORINFO_FLG_JIT_INTRINSIC)
```

Comment:
@AntonLapounov Can you please show me the place when I can disable pre-generation of the intrinsics bodies on Arm64 in CrossGen2?

Reply:
I think `CorInfoImpl.ShouldSkipCompilation` controls that.

Reply:
It looks it already does this right
https://github.com/dotnet/runtime/blob/557c7aff0d63fbb1bf687be215ec1726625abc2a/src/coreclr/src/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs#L190-L193

Thanks!